lib/webpacker/file_loader.rb

```diff
@@ -7,9 +7,17 @@ class FileLoaderError < StandardError; end
   attr_accessor :data
 
   class << self
+    def exist?
+      instance.data.present? rescue false
```

Comment:
The instance or the data not being there isn't really exceptional or unexpected, so let's use try or & to invocate rather than exception handling.